Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputSystemProvider.cs

```diff
@@ -610,17 +609,22 @@ void RegisterActions(Configuration cfg)
             if (m_ScrollWheelAction != null && m_ScrollWheelAction.action != null)
                 m_ScrollWheelAction.action.performed += OnScrollWheelPerformed;
 
-            // When adding new one's don't forget to add them to UnregisterActions
+            // When adding new actions, don't forget to add them to UnregisterActions
 
-            if (m_InputActionAsset != null)
+            if (InputSystem.actions == null)
+            {
+                // If we've not loaded a user-created set of actions, just enable the UI actions from our defaults.
+                m_InputActionAsset.FindActionMap("UI", true).Enable();
```

Comment:
OK. Maybe we should only create the UI actions, instead of creating all and leaving the rest disabled? But I understand that this would make the code more error-prone and isn't necessary at all for the current fix.

Reply:
Yeah, that was my line of thinking - plus, this will need a fuller re-write in the future anyway to remove DefaultInputActions.